### PR TITLE
Created ball storage subsystem, yay. Fixed some errors along the way and made a todo for the control panel

### PR DESCRIPTION
**Pull request by p-casals**
you're welcome by the way – I didn't actually do the control panel

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -66,8 +71,20 @@ private void configureButtonActions() {
...
         
-        controlLiftButton.cancelWhenPressed(Command.controlDrop);
-        controlLiftButton.whenPressed(Command.controlLift.withTimeout(3));
+        int x = 0;
+        if (x == 0) {
+            controlLiftButton.toggleWhenPressed(robotCommands.controlLift.withTimeout(3));
+            x = 1;
+        }
+        else {
+            controlLiftButton.toggleWhenPressed(robotCommands.controlDrop.withTimeout(3));
```
Comment (LordSquishers) [inline src/main/java/frc/robot/RobotContainer.java]: this does not work- please remove it

File: src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java [hunk 2]
```diff
@@ -1,16 +0,0 @@
-package frc.robot.commands.auto.routines;
-
-
-import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
-import frc.robot.commands.auto.MoveCommand;
-import frc.robot.commands.auto.TurnCommand;
-import frc.robot.subsystems.Drivetrain;
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java]: this should not be deleted- we still need it

**Review by LordSquishers — CHANGES_REQUESTED** [review]
fix

File: src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java (hunk 2 above)
Comment (p-casals) [reply src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java]: how do i undelete it then

Comment (p-casals) [reply src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java]: duly undeleted

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (p-casals) [reply src/main/java/frc/robot/RobotContainer.java]: removed + changed

**Comment by p-casals** [comment]
ok patrick it should be done now

File: src/main/java/frc/robot/RobotCommands.java [hunk 3]
```diff
@@ -22,8 +22,9 @@
     public final Climber CLIMBER = new Climber();
     public final Shooter SHOOTER = new Shooter();
     public final  Intake INTAKE = new Intake();
-    public final ControlPanel CONTROL_PANEL = new ControlPanel();
+    public final ControlPanel CONTROL = new ControlPanel();
```
Comment (LordSquishers) [inline src/main/java/frc/robot/RobotCommands.java]: CONTROL_PANEL

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -64,7 +70,14 @@ private void configureButtonActions() {
 
         // CONTROL PANEL BUTTONS
         controlSpinButton.whenHeld(robotCommands.controlSpin);
-        controlLiftButton.whenPressed(new ControlLiftCommand(robotCommands.CONTROL_PANEL).withTimeout(3));
+        controlLiftButton.whenPressed(robotCommands.liftControlMaybe);
+
+
```
Comment (LordSquishers) [inline src/main/java/frc/robot/RobotContainer.java]: spacing?

File: src/main/java/frc/robot/commands/auto/routines/Auto.java [hunk 5]
```diff
@@ -22,7 +21,7 @@
     /**
      * Creates a new TestAutoTwo.
      */
-    public Auto(Drivetrain DRIVETRAIN) {
+    public Auto(RobotCommands COMMAND, Drivetrain DRIVETRAIN) {
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/auto/routines/Auto.java]: COMMANDS

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 6]
```diff
@@ -61,14 +62,17 @@ public void reversePrimary() {
         primaryPiston.set(DoubleSolenoid.Value.kReverse);
         hasClimbed = false;
     }
+    public final BooleanSupplier hasClimbedBooleanSupplier = () -> hasClimbed;
+    public final BooleanSupplier pistonUpSupplier = () -> pistonUp;
 
+    // TODO: the booleansuppliers are suspicious. perhaps fix them since they're not periodic?
```
Comment (LordSquishers) [inline src/main/java/frc/robot/subsystems/Climber.java]: what does this mean?

**Review by LordSquishers — APPROVED** [review]

**Review by LordSquishers — APPROVED** [review]

**Pull request merged**